.github/workflows/sub-build-docker-image.yml

```
@@ -49,6 +47,13 @@ on:
         description: 'The image digest to be used on a caller workflow'
         value: ${{ jobs.build.outputs.image_digest }}
 
+
+env:
+  FEATURES: ${{ inputs.features || vars.RUST_PROD_FEATURES }}
+  TEST_FEATURES: ${{ inputs.test_features || vars.RUST_TEST_FEATURES }}
```

Line 32 mentions that `TEST_FEATURES` should default to `RUST_TEST_FEATURES` and `RUST_PROD_FEATURES`:

```suggestion
  TEST_FEATURES: ${{ inputs.test_features || format('{0} {1}', vars.RUST_TEST_FEATURES, vars.RUST_PROD_FEATURES) }}
```